frontend/src/k8sTypes.ts

```diff
@@ -1,6 +1,12 @@
 import { K8sResourceCommon } from '@openshift/dynamic-plugin-sdk-utils';
 import { PodAffinity, NotebookContainer, PodToleration, Volume, ContainerResources } from './types';
 
+export enum KnownLabels {
+  DASHBOARD_RESOURCE = 'opendatahub.io/dashboard',
+  MODEL_SERVING_PROJECT = 'modelmesh-enabled',
+  DATA_CONNECTION_AWS = 'opendatahub.io/managed',
+}
```

Comment:
Went to just add this for the model serving case, and sorta slipped into the other two. Data Connections have a weird label and Dashboard is just our fetch-specific label.

No one should declare these magic strings in the code anywhere.